### PR TITLE
Variant table revamped

### PR DESCRIPTION
**Pull request by lsheta**
Mega changes to the variant name table. Tons of examples added per several conversations.

### CODE REVIEW

File: docs/model/variants/name.rst [hunk 1]
```diff
@@ -33,22 +33,70 @@ When curating the Variant Name field, the most specific Variant Name described b
...
+S65W (c.194C>G)                  *VHL*    Missense Variant                 A specific amino acid change where the precise nucleotide change is provided or ascertainable. Protein change p. notation is implied, other HGVS notation (e.g. ‘ g.’ or ‘c.’) used  when applicable. This  degree of resolution is typically used for germline variants.
+E709A and G719C                  *EGFR*   Missense Variant                 Multiple concomitant missense variants in the same gene (e.g. double mutants). Should not be used when patients with different mutations in the same gene are pooled and analyzed.
+V600E or V600K                   *BRAF*   Missense Variant, 
+                                          Copy Number Gain 
+                                          {sequence feature}               Complex combination between two different types of variants in the same gene (e.g. missense and copy number gain).  Boolean order of operations and parentheses should be used when needed.
```
[the remark this answers is not in the document]
Comment (kkrysiak) [reply docs/model/variants/name.rst]: Added a commit to fix this df61cc214085ce3a8b45fe0d1b50e88356a4d7f3

**Review by kkrysiak — COMMENTED** [review]
Made 2 commits to correct typos, grammar and minor issues identified during this review.

File: docs/model/variants/name.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (lsheta) [reply docs/model/variants/name.rst]: the two commits look good. Thanks for the corrections.

[the remark this answers is not in the document]
Comment (jsaliba10) [reply docs/model/variants/name.rst]: The table order and descriptions look good.  There are a few  in the variant type column that has {sequence feature} after them, like the deletion entries. Should that { } be there?

**Review by obigriffith — COMMENTED** [review]
I spun up my own sphinx build and read through the whole variant table. I made some minor punctuation and rst formatting fixes. But overall this looks good to me. Definitely a huge improvement over what is already live.

**Review by obigriffith — APPROVED** [review]
This time approving my review.

**Review by jsaliba10 — APPROVED** [review]
changes look good.

**Pull request merged**